timemachine/fe/rest/single_topology.py

```diff
@@ -148,7 +148,7 @@ def target_propers(self) -> dict[int, CanonicalProper]:
         return {
             idx: proper
             for (idx, proper) in self.candidate_propers.items()
-            if any(idx in self.rest_region_atom_idxs for idx in astuple(proper))
+            if any(idx in self.rest_region_atom_idxs for idx in proper.idxs)
```

Comment:
was this a bug fix or a nit?

Reply:
edit: missed the description, this was a bug fix.

Reply:
This is the bug fix. `astuple` includes the `_unsafe` field of `CanonicalProper`:

```
>>> astuple(CanonicalProper.from_idxs(1, 2, 3, 4))
(1, 2, 3, 4, True)
```

leading to unintended results in the filter because e.g.

```
>>> True in {1}
True
```